events/wc_cc_missions.txt

```diff
@@ -162,4 +174,575 @@ society_quest_event = {
+
+	option = {
+		name = ACCEPT
+		set_quest = mission_build_barrow_den
+		character_event = { id = WCCC.230 days = 30 }
+	}
+
+	option = {
+		name = DECLINE
+		custom_tooltip = { text = decline_quest_tooltip }
+
+		add_character_modifier = {
+			name = quest_cooldown_timer
+			hidden = yes
+			days = 1000
+		}
+
+		ai_chance = { factor = 0 }
+	}
+}
+#QUEST: Build hospital
+society_quest_event = {
+	id = WCCC.214
+	border = GFX_event_letter_frame_religion
+
+	desc = {
+		text = EVTDESC_WCCC.214
+	}
+	is_triggered_only = yes
+
+	option = {
+		name = ACCEPT
+		set_quest = mission_build_hospital
+	}
+
+	option = {
+		name = DECLINE
+		custom_tooltip = { text = decline_quest_tooltip }
+
+		add_character_modifier = {
+			name = quest_cooldown_timer
+			hidden = yes
+			days = 1000
+		}
+
+		ai_chance = { factor = 0 }
+	}
+}
+character_event = {
+	id = WCCC.220
+	is_triggered_only = yes
+	hide_window = yes
+	
+	trigger = { has_quest = mission_build_hospital }
+	
+	immediate = {
+		random_society_member = {
+			limit = {
+				same_society_as = ROOT
+				is_society_grandmaster = yes
+			}
+			character_event = {
+				id = WCCC.221
+			}
+		}
+	}
+}
+character_event = {
+	id = WCCC.221
+	is_triggered_only = yes
+	hide_window = yes
+
+	immediate = {
+		FROM = {
+			character_event = {
+				id = WCCC.222
+			}
+		}
+	}
+}
+letter_event = {
+	id = WCCC.222
+	desc = EVTDESC_WCCC.222
+	border = GFX_event_letter_frame_religion
+	
+	is_triggered_only = yes
+	
+	trigger = { has_quest = mission_build_hospital }
+	
+	option = {
+		name = EVTOPTA_WCCC.222
+		clr_quest = mission_build_hospital
+		add_society_currency_tiny_effect = yes
+		
+	}
+}
+character_event = {
+	id = WCCC.223
+	is_triggered_only = yes
+	hide_window = yes
+	
+	trigger = { has_quest = mission_theology }
+	
+	immediate = {
+		random_society_member = {
+			limit = {
+				same_society_as = ROOT
+				is_society_grandmaster = yes
+			}
+			character_event = {
+				id = WCCC.224
+			}
+		}
+	}
+}
+character_event = {
+	id = WCCC.224
+	is_triggered_only = yes
+	hide_window = yes
+
+	immediate = {
+		FROM = {
+			character_event = {
+				id = WCCC.225
+			}
+		}
+	}
+}
+letter_event = {
+	id = WCCC.225
+	desc = EVTDESC_WCCC.225
+	border = GFX_event_letter_frame_religion
+	
+	is_triggered_only = yes
+	
+	trigger = { has_quest = mission_theology }
+	
+	option = {
+		name = EVTOPTA_WCCC.225
+		clr_quest = mission_theology
+		add_society_currency_tiny_effect = yes
+		
+	}
+}
+character_event = {
+	id = WCCC.230
+	desc = EVTDESC_WCCC.230
+	picture = GFX_evt_ritual_scroll
+	border = GFX_event_normal_frame_religion
+	
+	is_triggered_only = yes
+
+	
+	option = {
+		name = EVTOPTA_WCCC.230
+		character_event = { id = WCCC.231 days = 30}
+	}
+}
+character_event = {
+	id = WCCC.231
+	desc = EVTDESC_WCCC.231
+	picture = GFX_evt_ritual_scroll
+	border = GFX_event_normal_frame_religion
+
+	is_triggered_only = yes
+
+
+	option = {
+		name = EVTOPTA_WCCC.231
+		custom_tooltip = { text = barrow_den_cost_200 }
```

Comment:
Just a suggestion, it could be replaced by...
```suggestion
		tooltip = { wealth = -200 }
```
This will show gold cost, but won't take it.

Reply:
Nah, in these event chain you can pay, but it is not necessary if you have high skills, so it will just only confuse the player.